src/coreclr/jit/emitarm64.cpp

```diff
@@ -3041,7 +3041,9 @@ emitter::code_t emitter::emitInsCode(instruction ins, insFormat fmt)
             {
                 canEncode = true;
             }
-            if (allow_MSL)
+
+            // MSL is only supported for 32-bit.
+            if (allow_MSL && size == EA_4BYTE)
```

Comment:
nit: coding conventions specify fully-parenthesized expressions
```suggestion
            if (allow_MSL && (size == EA_4BYTE))
```